js/app/src/api_docs/ParametersSnippet.svelte

```diff
@@ -5,7 +5,7 @@
 	export let is_running: boolean;
 	export let endpoint_returns: any;
 	export let js_returns: any;
-	export let current_language: "python" | "javascript";
+	export let current_language: "python" | "javascript" | "bash";
```

Comment:
there's a problem with some types for js and bash 

python: 
<img width="870" alt="Screen Shot 2024-06-04 at 4 27 48 PM" src="https://github.com/gradio-app/gradio/assets/9021060/d5aa0675-d548-4f5e-9bee-95d14812acd5">

js: 
<img width="873" alt="Screen Shot 2024-06-04 at 4 27 57 PM" src="https://github.com/gradio-app/gradio/assets/9021060/5c1137bd-cf22-4ffb-9ee6-be305d2f8252">

bash: 
<img width="873" alt="Screen Shot 2024-06-04 at 4 27 37 PM" src="https://github.com/gradio-app/gradio/assets/9021060/39be0c49-6654-4c9e-a065-f8000ad4ad26">

Reply:
Looked into this issue and its actually not related to this PR. For now, I've put in a temporary fix that if no type information is available, it prints `any` instead:

<img width="729" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/056d7842-5bd0-4ecf-95c9-bc35c675f36b">